hs_core/hydroshare/utils.py

```diff
@@ -433,11 +433,19 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id, set_to_private=False):
     tgt_coll = tgt_res.root_path
     for avu_name in avu_list:
         value = istorage.getAVU(src_coll, avu_name)
+        # possibly create private copy if requested
```

Comment:
Assuming that we have unit tests that test resource copy creation, does it now need to test that the resource from the access control point have the attribute ```public``` set correctly?